services/keycloak/index.html

```diff
@@ -1,5 +1,45 @@
 ---
 layout: index
-title: Aerogear Keycloak
+title: Identity Management
 section: mobile-services
 ---
+
+<div class="row">
+    <div class="col-md-2">
+        <img style="width: 100px; height: 100px;" src="/img/keycloak.png" />
+    </div>
+    <div class="col-md-8">
+        <h2 class="section-header">
+            Mobile Identity Management
+            <i class="fa fa-lock" aria-hidden="true"></i>
+        </h2>
+    </div>
+
+</div>
+<div class="row">
+    <h3>Overview</h3>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit
+        ante. Proin cursus sagittis vehicula. Nulla mattis id erat non sagittis. Nullam ut ligula venenatis, ultrices nulla
+        a, convallis sem. Interdum et malesuada fames ac ante ipsum primis in faucibus. Vestibulum tincidunt, erat quis tincidunt
+        bibendum, est sapien viverra erat, et dictum felis dolor vel nulla. Sed hendrerit tincidunt ligula id mollis. Nam
+        iaculis lectus risus, sed suscipit dui tempus vitae. Pellentesque nec ultrices risus, sit amet vehicula nisi. </p>
+</div>
+<div class="row">
+    <h3>Getting Started</h3>
+    <p>This guide will show you how to set up the identity service and consume it to protect your mobile application.
+        <a target="_blank" href="https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/provision-keycloak-apb.adoc">Geting started with identity management for mobile via Keycloak</a>
+    </p>
+</div>
+<div class="row">
+    <h3>Documentation</h3>
+    <a target="_blank" href="https://docs.aerogear.org" class="btn btn-primary">
+        <i class="fa fa-book"></i> Guides! Docs! Learn more</a>
+</div>
+<div class="row">
+    <h3>Github</h3>
+    <a target="_blank" href="https://github.com/aerogearcatalog/keycloak-apb/" class="btn btn-primary btn-sm">
```

Comment:
Can the `btn-sm` be removed from the github icons? Might look better when it's the same size as the docs button